specs/enable_gluster_native_events.adoc

```diff
@@ -0,0 +1,147 @@
+= Enable gluster native events in tendrl
+
+Events generated by glusterfs events API should be exposed by tendrl.
+
+
+== Problem description
+
+Currently tendrl is not utilizing the events exposed by glusterfs events API.
+Tendrl should be enhanced to consume and handle the events.
+
+== Use Cases
+
+* Glusterfs events API pushes events for certain internal glusterfs failure conditions
+which is not track able by tendrl. Tendrl can raise suitable alerts for such events.
+
+* The interval of sds state sync can be increased (to 30 min) and we should have event
+driven state sync. which will considerably increase the efficiency of gluster integration
+service.
+
+== Proposed change
+
+The event receiver web-hook can be run on all the gluster storage nodes as a
+thread/greenlet in gluster-integration daemon. And we should configure the
+gluster eventing daemon to push events in localhost mode (the events generated
+by gluster eventing daemon will be pushed to the web-hook running only on
+that particular node). With this we are eliminating single point of failure
+w.r.t event listener.
+
+Since some of the events are raised from all the nodes or raised multiple times
```

Comment:
No we update the event to etcd, only if the state that is received from event is different from the state in etcd and we update the latest state to etcd. Thus in case of multiple duplicate events only the first one is saved in etcd